### PR TITLE
fix(GlobalHeader): Change focus outline color and add export

### PR DESCRIPTION
**Pull request by katiezutter**
### Overview

Changes GlobalHeader components focus outline to navy and add GlobalHeader to gamut-labs exports.

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

File: packages/gamut-labs/src/experimental/AppHeader/index.tsx [hunk 1]
```diff
@@ -24,6 +26,14 @@ export type AppHeaderItemsProp = {
   right: AppHeaderItem[];
 };
 
+const AppHeaderTextButton = styled(TextButton)`
+  ${focusStyles}
+`;
+
+const AppHeaderFillButton = styled(FillButton)`
+  ${focusStyles}
+`;
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/experimental/AppHeader/index.tsx:35]: ⛳ ✨ :

```ts
const AppHeaderTextButton = styled(TextButton)(focusStyles);

const AppHeaderFillButton = styled(FillButton)(focusStyles);
```

Explanation: string template literals (<code>name`stuff`</code>) are a fancy way of calling a function.

**Review by JoshuaKGoldberg — APPROVED** [review]
LGTM!

File: packages/gamut-labs/src/experimental/AppHeader/index.tsx (hunk 1 above)
Comment (katiezutter) [reply packages/gamut-labs/src/experimental/AppHeader/index.tsx:35]: 🤯

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.13.3-alpha.45aa9a.0<br/>@codecademy/styleguide@21.20.1-alpha.45aa9a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://600f31f078923f2cbf8f38c0--gamut-preview.netlify.app>https://600f31f078923f2cbf8f38c0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/600f31f078923f2cbf8f38c0>Deploy Logs</a></p>

File: packages/gamut-labs/src/experimental/AppHeader/index.tsx (hunk 1 above)
Comment (saghdaey) [reply packages/gamut-labs/src/experimental/AppHeader/index.tsx]: so fresh so clean

**Review by saghdaey — APPROVED** [review]
yes

**Pull request merged**